scep/scep.go

```diff
@@ -217,6 +224,11 @@ func ParsePKIMessage(data []byte, opts ...Option) (*PKIMessage, error) {
 		return nil, err
 	}
 
+	// According to RFC #2315 Section 9.1, it is valid that the server sends fewer
+	// certificates than necessary, if it is expected that those verifying the
+	// signatures have an alternate means of obtaining necessary certificates.
+	// In SCEP case, an alternate means is to use GetCaCert request.
+	p7.Certificates = append(p7.Certificates, conf.caCerts...)
```

Comment:
I have another proposal, please tell me WDYT about it.
I changed the above line such that if CA certificates are specified by ParsePKIMessage then we will use those as the only source of truth. That's assuming that there will be no use case in which a caller would want to add some source of truth but will also be ok if other signers exist.
IIUC, it is ok to do that in scepclient without a flag, because the scepclient in this implementation behaves as follows:
- Requests GetCaCert
- Use the CA certs retrieved to sign a CSR and sends it
- Receives a the response
At this point, won't the scepclient always be expecting a response only from one of the certs returned by the first step?
WDYT?